src/main/io/osd.c

```diff
@@ -600,6 +599,20 @@ static bool osdDrawSingleElement(uint8_t item)
 
             break;
         }
+		
+    case OSD_MOTOR_DIAG:
+        if( areMotorsRunning() ) {
+            float motorSum = 0;
+            int maxIdx = 0;
+            for( int i = 0; i < getMotorCount(); i++) {
+                if( motor[i] > motor[maxIdx] )
```

Comment:
Please always use a block `{...}` after `if` and `for`, even if it's just one statement.